WORKSPACE

```diff
@@ -14,6 +14,7 @@
 #
 ################################################################################
 #
+workspace(name = "proxy")
```

Comment:
Nit: `io_istio_proxy` (Bazel tries to use the reverse domain name notation).

Reply:
Done